server/channels/app/notification_push.go

```diff
@@ -107,32 +97,36 @@ func (a *App) sendPushNotificationToAllSessions(msg *model.PushNotification, use
 
 	if rejectionReason != "" {
 		// Notifications rejected by a plugin should not be considered errors
-		a.NotificationsLog().Info("Notification rejected by plugin",
-			mlog.String("type", model.TypePush),
-			mlog.String("status", model.StatusNotSent),
-			mlog.String("reason", rejectionReason),
+		// This is likely normal operation so no need for metrics here
```

Comment:
Maybe we can add a new `CountNotificationReason` "plugin"? 0/5

Reply:
Exposing this to plugin wouldn't necessarily be the worst idea actually...but we'll think about that going forward, if plugins plan to use it that is.